src/components/Label/Label.stories.mdx

```diff
@@ -52,6 +63,18 @@ import { getCategory } from "../../utils/componentCategories";
 
 <ArgsTable story="Label with Controls" />
 
+## Accessibility
```

Comment:
Should you mention here that there is no need to use this component in conjunction with any of the other Reservoir input element because those components already handle the a11y labelling?

Reply:
Yes.